crates/ruma-common/src/time.rs

```diff
@@ -28,6 +30,16 @@ impl MilliSecondsSinceUnixEpoch {
     pub fn to_system_time(self) -> Option<SystemTime> {
         UNIX_EPOCH.checked_add(Duration::from_millis(self.0.into()))
     }
+
+    /// Get the unix epoch in milliseconds.
+    pub fn get(&self) -> UInt {
+        self.0
+    }
+
+    /// Get the unix epoch in seconds.
+    pub fn as_secs(&self) -> UInt {
+        self.0 / UInt::from(MILLIS_PER_SEC)
```

Comment:
I think 1000 is not such a magic constant that it should be in a const. I'd suggest

```suggestion
        self.0 / uint!(1000)
```